models/edr/alerts/alerts_dbt_models.sql

```diff
@@ -6,9 +6,40 @@
 }}
 
 with error_models as (
-    select * from {{ ref('model_run_results') }}
+  
+    select model_execution_id,
```

Comment:
The original select was `select *` and it contains more columns 🙂 
To make sure we don't break anything that might be relaying on that model, lets add the rest of the columns:
```
model_execution_id,
unique_id,
invocation_id,
name,
generated_at,
status,
full_refresh,
message,
execution_time,
execute_started_at,
execute_completed_at,
compile_started_at,
compile_completed_at,
compiled_code,
database_name,
schema_name,
materialization,
tags,
package_name,
path,
original_path,
owner,
alias
```

Reply:
Updated